blog/2024-11-27-release-comms-data-link/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+slug: comms-nov2024
+title: |
+  🎉 Highlights from the Latest RenkuLab Releases! 🚀
+authors: [laura]
+tags:
+  - Release
+  - Renku 2.0
+  - Data Connector
+image: ./image_1.png
+description: Reuse data connectors across projects and bring your own image into Renku
+---
+
+![Renku Data Connectors](./image_1.png)
+
+<!-- truncate -->
```

Comment:
I really think you should truncate after the first paragraph here.

Reply:
![Screenshot 2024-11-28 at 09 21 24](https://github.com/user-attachments/assets/5d7d4dbc-40ec-4f84-b677-aa4b00319e31)

Reply:
Is it fine like this?

Reply:
👍